js/renovation/widget.tsx

```diff
@@ -213,7 +217,7 @@ export default class Widget extends JSXComponent<WidgetInput> {
                 e => !e.isDefaultPrevented() && (this._focused = true),
                 e => !e.isDefaultPrevented() && (this._focused = false),
                 {
-                    isFocusable: el => focusable(null, el),
+                    isFocusable: focusable,
```

Comment:
`focusable` method pass element as second parameter. Did this checked in IE?
In old component we used
```jsx
isFocusable: el => $(el).is(focusableSelector)
```
(that pass into function `index` and `element`, but we use only `element)
```js
focusable: function(index, element) {
  return focusable(element, $(element).attr('tabIndex'));
}
```